### PR TITLE
ROX-13420: Enable postgres on non-goovy test

### PR DESCRIPTION
**Pull request by c-du**
## Description

A few non-groovy tests has been disabled for various reasons. The original issue has been fixed.
https://github.com/stackrox/stackrox/pull/3685
But it looks like some non-groovy tests are out of sync. 

This PR enables the tests. I filed two jiras for active vuln and backup test which need more care to enable.

## Checklist
- [ ] Investigated and inspected CI test results
- ~[ ] Unit test and regression tests added~
- ~[ ] Evaluated and added CHANGELOG entry if required~
- ~[ ] Determined and documented upgrade steps~
- `[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

If any of these don't apply, please comment below.

## Testing Performed

TODO(replace-me)
Use this space to explain how you tested your PR, or, if you didn't test it, why
you did not do so. Valid reasons include, for example, "CI is sufficient",
"No testable changes". Feel free to attach JSON snippets, curl commands,
screenshots.

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

File: tests/roxctl/istio-support.sh [hunk 1]
```diff
@@ -58,12 +58,11 @@ test_roxctl_cmd() {
 
 test_roxctl_cmd central generate k8s none --output-format kubectl
 test_roxctl_cmd central generate openshift none
-if [ -z "${ROX_POSTGRES_DATASTORE}" ] || [ "${ROX_POSTGRES_DATASTORE}" == "false" ]; then #TODO(ROX-13420): Fix and enable on postgres
-    test_roxctl_cmd sensor generate k8s --name k8s-istio-test-cluster  --continue-if-exists
-    test_roxctl_cmd sensor get-bundle k8s-istio-test-cluster
-    test_roxctl_cmd sensor generate openshift --name os-istio-test-cluster --continue-if-exists
-    test_roxctl_cmd sensor get-bundle os-istio-test-cluster
-fi
+echo "[Warning] "
```
Comment (c-du) [inline tests/roxctl/istio-support.sh:61]: Will remove. Just want to let tests finish.

**Review by dashrews78 — APPROVED** [review]
LGTM

**Pull request merged**